app/gui/docs/product/visualizations.md

```diff
@@ -201,24 +200,11 @@ In particular:
   as visualizations. The superclass defines a default constructor and a set of
   utilities:
 
-  - #### Method `setPreprocessorCode(code)`
-    Set an Enso code which will be evaluated on the server-side before sending
-    data to visualization. If not called, a default unspecified code is used
+  - #### Method `setPreprocessor(module,method)`
+    Set an Enso method which will be evaluated on the server-side before sending
+    data to visualization. If not called, a default unspecified method is used
```

Comment:
_default unspecified_? That's a strange formulation in a **specification** of the behavior ;-)

Reply:
Why? Specification explicitly opts-out from defining this. 
There'll be some code present, but custom visualization should not make any assumptions about it. Thanks to that, we can adjust it without breaking public API.